frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+
+import {
+  // BaremetalHostRole,
+  // BaremetalHostStatus,
+  getName,
+  getNamespace,
+  // getResource,
+} from 'kubevirt-web-ui-components';
+
+// TODO(jtomasek): update import once models are moved to console-shared package
+// import { MachineModel, NodeModel } from '@console/internal/models';
+// import { referenceForModel } from '@console/internal/module/k8s';
+
+import {
+  ListHeader,
+  ColHead,
+  List,
+  ListPage,
+  ResourceRow,
+} from '@console/internal/components/factory';
+
+import { ResourceLink } from '@console/internal/components/utils';
+// import { WithResources } from '@console/shared';
+
+import { BaremetalHostModel } from '../models';
+import { getHostBMCAddress } from '../selectors';
+import MachineCell from './machine-cell';
+
+// const nameColumnClasses = 'col-lg-2 col-md-4 col-sm-6 col-xs-6';
+// const statusColumnClasses = 'col-lg-2 col-md-4 hidden-sm hidden-xs';
+// const machineColumnClasses = 'col-lg-3 visible-lg';
+// const roleColumnClasses = 'col-lg-2 visible-lg';
+// const addressColumnClasses = 'col-lg-2 visible-lg';
+const columnClasses = 'col-sm-4';
+
+const HostHeader = (props: React.ComponentProps<typeof ColHead>) => (
+  <ListHeader>
+    <ColHead {...props} className={columnClasses} sortField="metadata.name">
+      Name
+    </ColHead>
+    {/* <ColHead {...props} className={statusColumnClasses}>
+      Status
+    </ColHead> */}
+    <ColHead {...props} className={columnClasses}>
+      Machine
+    </ColHead>
+    {/* <ColHead {...props} className={roleColumnClasses}>
+      Role
+    </ColHead> */}
+    <ColHead {...props} className={columnClasses} sortField="spec.bmc.address">
+      Management Address
+    </ColHead>
+  </ListHeader>
+);
+
+const HostRow = ({ obj: host }: React.ComponentProps<typeof ResourceRow>) => {
```

Comment:
I had to switch from using React.FC as it did not satisfy react/prop-types lint rule.
(getting `error  'obj' is missing in props validation  react/prop-types`)